charts/cryostat/templates/_openshiftOauthProxy.tpl

```diff
@@ -41,6 +40,8 @@ Create OpenShift OAuth Proxy container.
   ports:
     - containerPort: 4180
       protocol: TCP
```

Comment:
The port `8443` is exposed anyway but I guess its good practice to also include in here too?